frontend/src/components/EstimateSection/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button } from '../shared/button';
+import { useEffect, useState } from 'react';
+import { API } from '../../modules/apis';
+
+const EstimateSection = () => {
+  const [data, setData] = useState({});
+
+  useEffect(() => {
+    API('/estimatesection').then(({ data: estimatesection }) => {
+      setData(estimatesection);
+    });
+  });
```

Comment:
no need to do this `({data: estimatesection})`, just keep it `data` and pass it to the state like this `setData(data)`

**and please add the brackets `[]` before the useEffect closing**

> because it caused to infinity rerender

useEffect should be like this 
`useEffect(() => { //your code here }, []);`